src/consensus/raft/rafttypes.h

```diff
@@ -109,8 +109,8 @@ namespace raft
   struct RequestVote : RaftHeader
   {
     Term term;
-    Index last_log_idx;
-    Term last_log_term;
```

Comment:
Strange to keep last for one but not the other, they are both conceptually last. Also it would be really good to put the vanilla raft names in comment next to them (“last_log_idx in normal raft, but last_commit_idx here to preserve verifiability” or something like that)

Reply:
Good point I'll do that